lib/LLVMIRCodeGen/BundleSaver.cpp

```diff
@@ -219,20 +219,20 @@ void BundleSaver::emitBundleEntryFunction() {
 // so that they know how much memory they need to allocate, etc.
 // Config consists of the following fields:
 // struct BundleConfig {
-//   size_t constantWeightVarsMemSize;
-//   size_t mutableWeightVarsMemSize;
-//   size_t activationsMemSize;
-//   size_t alignment;
-//   size_t numSymbols;
+//   uint64_t constantWeightVarsMemSize;
+//   uint64_t mutableWeightVarsMemSize;
+//   uint64_t activationsMemSize;
+//   uint64_t alignment;
+//   uint64_t numSymbols;
 //   SymbolTableEntry *symbolTable;
 // };
 void BundleSaver::emitBundleConfig() {
   auto symbolTable = irgen_->getModule().getGlobalVariable(
       irgen_->getMainEntryName() + "SymbolTable", true);
   GLOW_ASSERT(symbolTable &&
               "Expected to find a symbol table for the AOT bundle");
-  // Get the integer type having the same size in bits as size_t.
-  auto *SizeTType = irgen_->getBuilder().getIntNTy(sizeof(size_t) * 8);
+  // Get the integer type having the same size in bits as uint64_t.
+  auto *SizeTType = irgen_->getBuilder().getIntNTy(sizeof(uint64_t) * 8);
```

Comment:
The name `SizeTTy` doesn't make a lot of sense if we use uint64_t now. Also the name should start with a lower-case.

Reply:
Does it make more sense to use the newly introduced `LLVMIRGen::getTargetSizeTWidth`?

Reply:
@petecoup If we decide to go for target-specific `size_t` usage, we would definitely use `LLVMIRGen::getTargetSizeTWidth`. 

We are still trying to decide if it is better to use fixed-size fields instead, so that the `BundleConfig` has the same binary layout on all platforms and can be easily read e.g. by tools, helper functions, etc without thinking about any target-specific aspects.

Reply:
Sorry, I somehow missed the later comments...this makes sense!